### PR TITLE
Ccl dbmemory

### PR DESCRIPTION
**Pull request by catlook**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-f82784ae45/).

File: _posts/2018-06-20-array-diagnostic-utility-reports-in-vmware-esxi.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
```
Comment (cyrichardson) [inline _posts/2018-06-20-array-diagnostic-utility-reports-in-vmware-esxi.md]: I think this file can be deleted. It's in PR #303.

Comment (catlook) [reply _posts/2018-06-20-array-diagnostic-utility-reports-in-vmware-esxi.md]: still needs your edits

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-5315326ea0/). :bow:

**Comment by cyrichardson** [comment]
@catlook Suggested changes here: 
[blog_Oracle database In-memory Advisor_cyrEdits.docx](https://github.com/rackerlabs/docs-developer-blog/files/2115817/blog_Oracle.database.In-memory.Advisor_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-7ebf931216/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-7ebf931216/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-f1e0de062c/)

**Pull request merged**